src/main/java/ch/njol/skript/expressions/ExprYawPitch.java

```diff
@@ -55,7 +66,9 @@ public boolean init(final Expression<?>[] exprs, final int matchedPattern, final
 	}
```

Comment:
Can't request changes on `init` method but the final modifier on method params should be removed as well